src/server/game/Entities/Player/Player.cpp

```diff
@@ -6367,6 +6367,15 @@ void Player::SendMessageToSetInRange(WorldPacket const* data, float dist, bool s
     Cell::VisitWorldObjects(this, notifier, dist);
 }
 
+void Player::SendChatMessageToSetInRange(WorldPacket const* data, float dist, bool self, bool own_team_only) const
```

Comment:
maybe this method should be called like all the other overloads SendMessageToSetInRange() and have 1 additional parameter about 3d distance ?

Reply:
I guess. so i'll add last default bool argument to the SendMessageToSetInRange() instead a new method creation

Reply:
just avoided to change public method and added a private new instead 
If it's ok to change public method I will

Reply:
In general it's ok to change any public method as long as all the references in TC are updated to fix the build.
The only case where we might want to discuss is AI and script-related hooks that might break custom scripts by users, but even then it's up to users to fix their custom code.

So please feel free to change public methods.

Reply:
Fixed in last commit